packages/model/src/discover.ts

```diff
@@ -5,6 +5,7 @@ import { NewsResponse } from './news';
 export interface DiscoverResponse {
   aboutUs: string;
   members: ReadonlyArray<UserResponse>;
+  membersHref?: string;
```

Comment:
I would propose to return an ID of the ASAP team instead
```suggestion
  asapTeamId?: string;
```